package.json

```diff
@@ -98,6 +99,7 @@
     "karma-jasmine": "~3.3.0",
     "karma-jasmine-html-reporter": "^1.5.0",
     "karma-junit-reporter": "^1.2.0",
+    "license-checker": "^25.0.1",
```

Comment:
Why did you add license-checker, there are no changes in the demo for license or an automated use at buildtime.

Reply:
That's indeed superfluous. Removed! Thanks!